demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -715,21 +715,22 @@ def validate_no_duplicated_release_notes(self, added_files):
         click.secho("\nNo duplicated release notes found.\n", fg="bright_green")
         return True
 
-    def validate_no_missing_release_notes(self, modified_files, old_format_files, added_files):
+    def validate_no_missing_release_notes(self, modified_files, old_format_files, added_files, changed_meta_files):
         """Validate that there are no missing RN for changed files
 
         Args:
             modified_files (set): a set of modified files.
             old_format_files (set): a set of old format files that were changed.
             added_files (set): a set of files that were added.
+            changed_meta_files (set): a set of modified metadata files.
 
         Returns:
             bool. True if no missing RN found, False otherwise
         """
         click.secho("\n================= Checking for missing release notes =================\n", fg="bright_cyan")
 
         # existing packs that have files changed (which are not RN, README nor test files) - should have new RN
-        changed_files = modified_files.union(old_format_files).union(added_files)
+        changed_files = modified_files.union(old_format_files).union(added_files).union(changed_meta_files)
```

Comment:
From this, it looks like we weren't considering changes for metadata files when checking for RN. Isn't this contradict the issue?